application-templates/starter/src/components/channel-details/transform-errors.js

```diff
@@ -2,6 +2,18 @@ import omitEmpty from 'omit-empty-es';
 
 const DUPLICATE_FIELD_ERROR_CODE = 'DuplicateField';
 
+const setFieldValidationError = ({
+  formErrors,
+  fieldName,
+  validationError,
+}) => {
+  if (fieldName in formErrors) {
+    formErrors[fieldName][validationError] = true;
```

Comment:
I'm not sure if you're ok with this kind of `formErrors` object mutation inside a helper function